Source/OxyPlot/PlotModel/PlotModel.Rendering.cs

```diff
@@ -127,6 +127,7 @@ protected virtual void RenderOverride(IRenderContext rc, double width, double he
                             "An exception of type {0} was thrown when rendering the plot model.\r\n{1}",
                             exception.GetType(),
                             exception.GetBaseException().StackTrace);
+                    this.updateException = exception;
```

Comment:
Wouldn't it better to create a new field `renderException`? And a `GetLastRenderException()` method? Or rename to `lastException` and `GetLastException()`?